drivers/mfd/Kconfig.adi_max6639

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2025 Tenstorrent AI ULC
+# SPDX-License-Identifier: Apache-2.0
+
+config MFD_ADI_MAX6639
+	bool "ADI MAX6639 PWM Fan-Speed Controller, MFD device"
+	default y
+	depends on DT_HAS_ADI_MAX6639_ENABLED
+	depends on I2C
+	help
+	  This option enables the Analog Devices max6639 PWM fan controller.
+
```

Comment:
Not sure if it's customary to surround this in an `if`. Just something I noticed, but not blocking.